pkg/engine/secrets/inspector.go

```diff
@@ -177,12 +179,28 @@ func (c *Inspector) Inspect(ctx context.Context, basePaths []string,
 	return c.vulnerabilities, nil
 }
 
-func compileRegexQueries(queryFilter *source.QueryInspectorParameters, allRegexQueries []RegexQuery) ([]RegexQuery, error) {
+func compileRegexQueries(queryFilter *source.QueryInspectorParameters, allRegexQueries []RegexQuery, isCustom bool) ([]RegexQuery, error) {
 	var regexQueries []RegexQuery
+	var includeSpecificSecretQuery bool
+
+	allSecretsQueryAndCustom := false
+
+	includeAllSecretsQuery := isValueInArray("a88baa34-e2ad-44ea-ad6f-8cac87bc7c71", queryFilter.IncludeQueries.ByIDs)
```

Comment:
I'd suggest getting the query ID from here https://github.com/Checkmarx/kics/blob/master/assets/assets.go#L12